### PR TITLE
Add go-env

### PR DESCRIPTION
**Pull request by kvandenhoute**
Alternative for #17

### CODE REVIEW

File: internal/replicator/k8s.go [hunk 1]
```diff
@@ -273,6 +274,16 @@ func getSecretOnLabel(uuid string, namespace string) (v1.Secret, error) {
 	return secretList.Items[0], nil
 }
 
+func getCurrentNamespace() string {
```
Comment (kvandenhoute) [inline internal/replicator/k8s.go:278]: maybe we can do something like 
if (env.namespace set; use this)
else if(getcurrentnamespace works; use this)
else use dev-tooling (or smth else)

File: go.mod [hunk 2]
```diff
@@ -3,15 +3,18 @@ module github.com/kvandenhoute/sofplicator
 go 1.18
```
Comment (stgrace) [inline go.mod:3]: permissions on this file changed to 755

File: go.mod [hunk 3]
```diff
@@ -96,7 +106,7 @@ require (
 require (
 	github.com/Azure/azure-sdk-for-go/sdk/azidentity v1.2.0
 	github.com/Azure/azure-sdk-for-go/sdk/keyvault/azsecrets v0.11.0
-	github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/keyvault/armkeyvault v1.0.0
 	github.com/gin-gonic/gin v1.8.1
-	golang.org/x/sys v0.0.0-20220722155257-8c9f86f7a55f // indirect
+	github.com/spf13/viper v1.14.0
```
Comment (stgrace) [inline go.mod]: Don't think we are using viper?

File: internal/replicator/k8s.go (hunk 1 above)
Comment (stgrace) [reply internal/replicator/k8s.go:278]: Sounds good to me

**Review by stgrace — COMMENTED** [review]
LGTM except the go.mod

**Review by stgrace — COMMENTED** [review]
Good stuff

**Review by stgrace — APPROVED** [review]

**Pull request merged**